Dag37.RabbitFramework/RabbitFramework.Test/RabbitBusProviderTests.cs

```diff
@@ -122,29 +122,29 @@ public void BasicPublishThrowsArgumentExceptionWhenMessageIsNull()
         [TestMethod]
         public void SetupRpcListenerThrowsArgumentExceptionWhenQueueNameIsNull()
         {
-            var exception = Should.Throw<ArgumentException>(() => _sut.SetupRpcListener(null, _commandReceivedCallback));
-            exception.Message.ShouldBe(QueueNameParamName);
+            var exception = Should.Throw<ArgumentException>(() => _sut.SetupRpcListeners(null, null, _commandReceivedCallback));
+            exception.Message.ShouldBe("The queue name should not be null");
         }
 
         [TestMethod]
         public void SetupRpcListenerThrowsArgumentExceptionWhenQueueNameIsEmpty()
         {
-            var exception = Should.Throw<ArgumentException>(() => _sut.SetupRpcListener("", _commandReceivedCallback));
-            exception.Message.ShouldBe(QueueNameParamName);
+            var exception = Should.Throw<ArgumentException>(() => _sut.SetupRpcListeners("", null,_commandReceivedCallback));
```

Comment:
Hier kan je beter een ArgumentNullException gooien, gezien het om null parameters gaat. Let op: het is hier best-practice om de parameter naam aan de ArgumentNullException mee te geven ('nameof(ding)'). In de assert moet je even opletten dat je de exception.Parameter pakt ipv de exception.Message (heeft even geduurd voordat ik hierachter was ;p)